package.json

```diff
@@ -95,6 +95,7 @@
     "gh-pages": "^2.0.1",
     "glob": "^7.1.2",
     "jest": "^26.6.3",
+    "ky": "^0.12.0",
```

Comment:
This was added because a different version of it was getting installed in the root node_modules due to `terra-functional-testing -> webdriverio -> webdriver`. This was causing the danger command to fail in Travis.

Reply:
interesting, thanks for noting why it was added.